src/test/end-to-end/tests/system-admin/system-admin-user-management-test.ts

```diff
@@ -0,0 +1,133 @@
+import { config as testConfig } from '../../../config';
+import { randomData } from '../../shared/random-data';
+
+Feature('System admin User Management');
+
+const testFirstName = 'System Admin Test First Name';
+const testLastName = 'System Admin Test Surname';
+
+const testEmailAddress = 'pip-e2e-test-admin-management-' + randomData.getRandomNumber(1, 10000) + '@hmcts.net';
+const systemAdminUsername = testConfig.SYSTEM_ADMIN_USERNAME as string;
+
+Scenario('I as a system admin should be able to update a users role and delete a user', async ({ I }) => {
+    I.loginAsSystemAdmin();
+    I.click('Admin Dashboard');
+    I.createAdminAccount(testFirstName, testLastName, testEmailAddress, 'Internal - Administrator - Local');
+    I.click('Home');
+    I.see('System Admin Dashboard');
+    I.click('#card-user-management');
+    I.see('User Management');
+    I.fillField('#email', testEmailAddress);
+    I.click('Apply filters');
+    I.see(testEmailAddress);
+    I.click('#manage-link');
+    I.see('Manage ' + testEmailAddress);
+    I.click('Change');
+    I.selectOption('#updatedRole', 'CTSC Admin');
+    I.click('Continue');
+    I.see('User Updated');
+    I.see('This user has been updated to a CTSC Admin');
```

Comment:
When the text is result of an action then first time we use I.waitForText
  I.click('Continue');
    I.see('User Updated');
    I.see('This user has been updated to a CTSC Admin');